packages/discord-bot/src/handlers/praise.ts

```diff
@@ -67,6 +68,9 @@ export const praiseHandler: CommandHandler = async (
   }
 
   const userAccount = await getUserAccount(member as GuildMember);
+  const praiseItems = await PraiseModel.find({
```

Comment:
Q) Wouldn't this become an expensive call?
Could we send this message to users when they activate their praise account instead of sending it on first praise?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
We don't have any huge performance concerns here since praising is relatively infequent. But yes, this does seem like an overly expensive call. To begin with I'd suggest we try doing `PraiseModel.countDocuments(` instead.

Reply:
that should make this more feasible 👍 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good